features/step_definitions/index_page_steps.rb

```diff
@@ -1,3 +1,8 @@
 Given(/^I am on the index page$/) do
   visit root_path
 end
+
+And(/^(?:I expect a Google map to load|the map has been loaded)$/) do
```

Comment:
You are using the | to be able to use both texts in your tests. So if you were going to continue testing this map in different contexts and scenarios it would be a good idea to use this | pipe thing. If you change the thing me and thomas commented on in the tests this should only contain "I expect a Google map to load"